doc/EPIC.md

```diff
@@ -0,0 +1,110 @@
+- On the interface-pairs (ingress/egress pair) that affect the hidden path, the last two ASes either:
+  1. Only allow EPIC-HP path type traffic. See use case "Highly Secure Hidden Paths" [here](#HighlySecureHiddenPaths). The path type filtering is further explained [here](#PathTypeFiltering).
+  2. Prioritize EPIC-HP path type traffic. See use case "DOS-Secure Hidden Paths" [here](#DOSSecureHiddenPaths).
+- The last two ASes of the hidden path have a duplicate-suppression system in place. This prohibits DOS attacks based on replayed packets.
+
+### Example
+The following figure illustrates those assumptions:
+<p align="center">
+  <img src="fig/EPIC/path-type-filtering.png" width="630">
+</p>
+
+Here, AS 6 is the AS protected by the hidden path (blue lines). The hidden path ends at the local interface (black dot) of AS 6, so AS 6 did not forward the beacon that defines the hidden path further down to AS 7. 
+This is however still allowed for SCION path type traffic (green lines): there are SCION paths that enter AS 6 from AS 4. One of the two paths ends in the local interface of AS 6, while the other one is extended further to AS 7. 
+
+In this example, the border routers of AS 6 and AS 5 (the last and penultimate ASes on the hidden path) further implement path type filtering (red dots). Metaphorically speaking , they do not allow green lines to overlap with blue lines. For example, AS 5 will block (red "X" in the figure) SCION path type traffic from AS 3 that is destined towards AS 6, as it would affect the hidden path. Instead of blocking non-EPIC-HP path type traffic, ASes could also prioritize EPIC-HP traffic, which would still satisfy the assumptions above.
+
+Of course the ASes can always decide to be more restrictive, for example AS 6 could additionally disallow SCION path type traffic from AS 4, so that its local interface is reachable through the hidden path only.
+
+### SCION Path Type Responses
+EPIC-HP path type packets contain the full SCION path type header plus a timestamp and verification fields for the penultimate and last ASes on the path. 
+
+The included SCION path type header allows the destination behind a hidden path to directly respond with SCION path type packets. The destination only has to extract the SCION path type header from the EPIC-HP header and reverse the path. For this, there is a "SCION-Response (S)" flag in the EPIC-HP path type header, which can be set by the source. Setting this flag (S = 1) makes sense if the source AS is not behind a hidden path, or if it is behind a hidden path but also allows SCION path type traffic (but with lower priority than EPIC-HP).
+
+If the SCION-Response flag is not set (S = 0), this means that the source is itself behind a hidden path. The destination will therefore answer with a new EPIC-HP packet, provided it has the necessary authenticators for the hidden path towards the source.
+
+<p align="center">
+  <img src="fig/EPIC/SCION-reponse-flag.png" width="600">
```

Comment:
Why are there multiple green response arrows but only one blue one?

Reply:
Changed it to multiple blue arrows.